gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=0.8.0-SNAPSHOT
+version=0.8.1-SNAPSHOT
```

Comment:
Let's make this 0.9.0-SNAPSHOT. Patch versions typically don't have snapshots

Reply:
Sure, done.